packages/salesforcedx-vscode-lwc/test/vscode-integration/commands/forceLightningLwcStart.test.ts

```diff
@@ -261,6 +261,19 @@ describe('forceLightningLwcStart', () => {
         );
       });
 
+      it('shows no error when server is stopping', () => {
```

Comment:
Added a new test to ensure that no errors were printed by the forceLightningLwcStart command when shutting down the server normally.